src/components/DetailsInfo/detailsInfo.util.js

```diff
@@ -206,13 +205,19 @@ export const generateAlertsDetailsInfo = pageData => {
       notificationsDetailsInfo: [],
       triggerCriteriaDetailsInfo: []
     }
-    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((valueItem, index) => (
-      <Tooltip
-        key={valueItem.tooltip + index}
-        template={<TextTooltipTemplate text={valueItem.tooltip} />}
-      >
-        {valueItem.icon}
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item_header">{upperFirst(notification.kind)}</div>
```

Comment:
we add only modifiers with "_", I believe "header" is not a modifier

Reply:
Done